cernopendata/modules/fixtures/data/records/cms-detailed-luminosity-information.json

```diff
@@ -70,7 +70,7 @@
value is available for a particular lumi section, then the <strong>hfoc</strong> (Hadron Forward Zero Counting) value should be retrieved for that lumi section.</p>\n<p>The integrated luminosity values for validated runs and luminosity sections of the 2012 RunA, RunB, RunC, and RunD data-taking periods for the two luminometers, separately, are available in the tables<ul><li><a href=\"/record/1054/files/2012RunAlumi_pxl.txt\">2012RunAlumi_pxl.txt</a></li><li><a href=\"/record/1054/files/2012RunAlumi_hfoc.txt\">2012RunAlumi_hfoc.txt</a><li><a href=\"/record/1054/files/2012RunBlumi_pxl.txt\">2012RunBlumi_pxl.txt</a></li><li><a href=\"/record/1054/files/2012RunBlumi_hfoc.txt\">2012RunBlumi_hfoc.txt</a></li><li><a href=\"/record/1054/files/2012RunClumi_pxl.txt\">2012RunClumi_pxl.txt</a></li><li><a href=\"/record/1054/files/2012RunClumi_hfoc.txt\">2012RunClumi_hfoc.txt</a><li><a href=\"/record/1054/files/2012RunDlumi_pxl.txt\">2012RunDlumi_pxl.txt</a></li><li><a href=\"/record/1054/files/2012RunDlumi_hfoc.txt\">2012RunDlumi_hfoc.txt</a></li></ul></p>\n<p>The integrated luminosity values of all 2012 p-p data taking in single tables, for the two luminometers, are available in<ul><li><a href=\"/record/1054/files/2012lumi_pxl.txt\">2012lumi_pxl.txt</a></li><li><a href=\"/record/1054/files/2012lumi_hfoc.txt\">2012lumi_hfoc.txt</a></li></ul></p>\n<p>Note that if there are <em>orphan</em> lumi sections &ndash;listed at the bottom of the lumi files as <em>in json but not in results</em>&ndash; for which there are neither pxl values nor hfoc values, they can be safely ignored.  These lumi sections have no events as they correspond to the very end of runs for which the <em>stable beam</em> flag was not disabled yet during acquisition.</p>\n<p>The uncertainty in the luminosity measurement of 2012 data should be considered as 2.6% (reference <a href=\"https://cds.cern.ch/record/1598864\">CMS PAS LUM-13-001</a>).</p>\n<p>In your estimate for the integrated luminosity, check for which runs the trigger you have selected is active and sum the values for those runs. If you are using prescaled triggers, you can find the trigger prescale factors as shown in <a href=\"/record/5004\">the trigger examples</a>. The change of prescales (run, lumi section, index of prescales) is logged in the <a href=\"/record/1052/files/prescale2012.txt\">prescale2012.txt</a> file from a <a href=\"/record/1052\">previous version</a> of this record.</p>\n<p>For luminosity calculation, a detailed list of luminosity by lumi section is provided in <a href=\"/record/1054/files/2012lumibyls_pxl.csv\">2012lumibyls_pxl.csv</a> and <a href=\"/record/1054/files/2012lumibyls_hfoc.csv\">2012lumibyls_hfoc.csv</a> for the <a href=\"/record/1002\">list of validated runs</a> and lumi sections.</p>\n<p>Additional information on how to extract luminosity values using the <strong>brilcalc tool</strong> can be found in the <a href=\"/docs/cms-guide-luminosity-calculation\"> luminosity calculation guide</a>.</p>"
```

Comment:
Note: this changed was already merged in #3292. I'll take the four files and enrich the record.